src/common/components/cards/failed-instances-section.tsx

```diff
@@ -20,7 +20,7 @@ export type FailedInstancesSectionProps = {
 export const FailedInstancesSection = NamedFC<FailedInstancesSectionProps>(
     'FailedInstancesSection',
     ({ cardsViewData, deps, userConfigurationStoreData, targetAppInfo, shouldAlertFailuresCount }) => {
-        if (cardsViewData.cards == null) {
+        if (cardsViewData == null) {
```

Comment:
Would we also want to catch the scenario where cardsViewData.cards is null?
if (cardsViewData == null || cardsViewData.cards == null)